### DISCUSSION TITLE
Check the user authentication microservice in GO

### DISCUSSION THREAD

**Discussion by mstomar698** [scarelma/IDE#14] [Ideas]
I just checked the microservices for the user login and signup system using go.
what abt it? Would u like to try it first or do I start react one?
check the docs [here](https://mattermost.com/blog/how-to-build-an-authentication-microservice-in-golang-from-scratch/) and git repo [here](https://github.com/shadowshot-x/micro-product-go) .
.....

**Comment by mukul-kr** [comment]
This is good but I think this provides API only, so you can go ahead and do a react implementation of the above.
